static/js/share.js

```diff
@@ -0,0 +1,14 @@
+function showOverlay() {
+  const overlay = document.getElementsByClassName('socialHolder')[0];
+  const shadow = document.getElementsByClassName('overlayDark')[0];
+  if (shadow.classList.contains("displayNone")) {
+    overlay.classList.remove("displayNone");
```

Comment:
You might be able to use `classList.toggle` instead of `classList.remove` and `classList.add`. This way you can remove the `else` statement

Reply:
I did do the Remove and add function instead of Toggle because toggle always gave me problems in the long run.